resources/definitions/3DI_D300_v2

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Please hold similar file naming conventions to the rest of the printers: lowercase with underscores.